plugins/eslint-plugin-liferay-portal/tests/lib/rules/no-global-fetch.js

```diff
@@ -45,6 +45,26 @@ ruleTester.run('no-global-fetch', rule, {
 				}
 			`,
 		},
+		{
+			// Unnamed import from '../fetch.es'
+			code: `
+				import fetch from '../fetch.es';
```

Comment:
I think we need a counter-example that should fail:

```javascript
import fetch from 'whatwg-fetch';
```
or:

```javascript
import fetch from 'isomorphic-fetch;
```

Reply:
Ah, yeah, that's what lead me into my own 🐰hole today. Initially, I added:

```javascript
import 'whatwg-fetch'
 
window.fetch(...)
```
 
which is the one in their docs... and which we let pass because of `window.` qualifier 😂 

So, I'll just add

```javascript
import fetch from 'whatwg-fetch';

fetch(url);
```